pkg/util/virtualmachineinstance/virtualmachineinstance.go

```diff
@@ -99,6 +99,13 @@ func ValidateVMMigratable(vmi *kubevirtv1.VirtualMachineInstance) error {
 		return fmt.Errorf("VM %s is not live migratable as CD-ROM or container disk is set", vmiNamespacedName)
 	}
 
+	// Lastly, check the condition reported by KubeVirt
+	for _, cond := range vmi.Status.Conditions {
+		if cond.Type == kubevirtv1.VirtualMachineInstanceIsMigratable && cond.Status == corev1.ConditionFalse {
```

Comment:
when the target condition is found, return error or nil, the loop can be finished; and also could you add a test code to cover this case? thanks.

Reply:
Sure. Sounds good!